### DISCUSSION TITLE
Copy text from the diff view

### DISCUSSION THREAD

**Discussion by cpholguera** [community/community#10096] [Mobile] [answered]
It'd be great to be able to copy from the Diff view. At least to copy the whole line (maybe a copy button next to the comment button?)

![Screenshot_20220113-210503](https://user-images.githubusercontent.com/29175115/149401614-6b8b1bee-0b0f-42fd-820d-7c7c3c6e7e8e.jpg)

**Comment by brianlovin** [comment, marked as answer]
Agreed, this is not ideal. I've added this discussion to our tracking issue so that we can prioritize the work in the future — thanks!
